proto/src/build.rs

```diff
@@ -2,6 +2,10 @@ use std::io::Result;
 
 fn main() -> Result<()> {
     std::env::set_var("PROTOC", protobuf_src::protoc());
-    prost_build::compile_protos(&["proto/test_context.proto"], &["proto/"])?;
+    let mut prost_build = prost_build::Config::new();
+    prost_build
+        .type_attribute(".", "#[derive(serde::Serialize,serde::Deserialize)]")
+        .extern_path(".google.protobuf.Timestamp", "::prost_wkt_types::Timestamp")
+        .compile_protos(&["proto/test_context.proto"], &["proto/"])?;
```

Comment:
This allows serialization / deserialization. We have to overwrite the google protobuf types with the corresponding prost types in order to get that.